pycon/templates/finaid/receipt_upload.html

```diff
@@ -0,0 +1,61 @@
+{% extends "site_base.html" %}
+
+{% load bootstrap_tags %}
+{% load compress %}
+{% load static %}
+{% load i18n %}
+{% load boxes_tags %}
+
+{% block head_title %}Receipt Upload{% endblock %}
```

Comment:
Let's go ahead and mark all the English text in the template for translation.  (Put `{% trans"` and `" %}` around it.)